File: 프로그래머스/lv0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    if price >= 500000:
        return int(price - price*0.2)
    if price >= 300000:
        return int(price - price*0.1)
    elif price >= 100000:
        return int(price - price*0.05)
    else:
        return price
 

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        if (price >= 500_000) {
            return (int) (price*0.8);
        }
          else if (price >= 300_000) {
            return (int) (price*0.9);
        }
          else if (price >= 100_000) {
            return (int) (price*0.95);
        }
         return price;
    }
}